contracts/fair-burn/src/msg.rs

```diff
@@ -0,0 +1,31 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+
+use crate::state::Config;
+
+#[cw_serde]
+pub struct InstantiateMsg {
+    /// The percentage of funds to be burned, represented as basis points
+    pub fee_bps: u64,
+}
+
+#[cw_serde]
+pub enum ExecuteMsg {
+    FairBurn { recipient: Option<String> },
+}
+
+#[cw_serde]
+#[derive(QueryResponses)]
+pub enum QueryMsg {
+    #[returns(ConfigResponse)]
```

Comment:
This could just return `Config` right? It doesn't necessarily need the `ConfigResponse` wrapper.